tests/integration-tests/tests/efa/test_efa/test_efa/pcluster.config.yaml

```diff
@@ -35,7 +35,8 @@ Scheduling:
             {% if instance == "p4d.24xlarge" %}GdrSupport: true{% endif %}
 {% if instance == "p4d.24xlarge" %}
           CapacityReservationTarget:
-            CapacityReservationResourceGroupArn: arn:aws:resource-groups:us-east-1:447714826191:group/EC2CRGroup
+            #CapacityReservationResourceGroupArn: arn:aws:resource-groups:us-east-1:447714826191:group/EC2CRGroup
+            CapacityReservationId:
```

Comment:
Missing ODCR ID

Reply:
ops I didn't want to commit this. REmoving